cinema/models.py

```diff
@@ -60,6 +60,10 @@ class Meta:
     def __str__(self):
         return self.movie.title + " " + str(self.show_time)
 
+    @property
+    def available_places(self):
+        return self.cinema_hall.capacity - self.tickets.count()
```

Comment:
The `available_places` property in the `MovieSession` class assumes that the `tickets` related name is correctly set in the `Ticket` model's `movie_session` ForeignKey. Ensure that the `related_name='tickets'` is indeed set in the `Ticket` model, which it is. Additionally, consider handling cases where `self.cinema_hall` or `self.tickets` might be `None` to avoid potential errors.